src/main/js/controls/Graph/Graph.js

```diff
@@ -377,6 +381,82 @@ class Graph extends Construct {
         return this;
     }
 
+    /**
+     * Updates the graph axisData and content.
+     *
+     * @returns {Graph} - Graph instance
+     * @param {Array} graphData - Input array that holds updated values and key
+     */
+    reflow(graphData) {
+        this.config.axis.x.domain = getDomain(
```

Comment:
Let's use the Single Responsibility design principle and move each updates to their own respective module. 
Axis updates to axis.js, Region updates to region.js, etc.

Reply:
I have created a function in `axis.js` on [line 812](https://github.com/cerner/carbon-graphs/pull/138/files#diff-c116b52d2491f4d3b18768a42e2a7a23R812). This function is being used to update the x axis domain in `Graph.js` on [line 391](https://github.com/cerner/carbon-graphs/pull/138/files#diff-4c7acbc816e711a56dc8fffdd18a4149R391). Is this what you meant? @abhijit945